src/Docker/JobDefinition.php

```diff
@@ -70,6 +78,14 @@ public function getState(): array
         return $this->state;
     }
 
+    /**
+     * @return ObjectEncryptor::BRANCH_TYPE_DEV|ObjectEncryptor::BRANCH_TYPE_DEFAULT
+     */
+    public function getBranchType(): string
+    {
+        return $this->branchType;
+    }
```

Comment:
Tady mi to prestava sedet
* IMHO tady uz by to nemely byt stringy, branch type by mel byt enum
* dava smysl, ze z JobDefinition vracis nejaky ObjectEncryptor-scoped hodnoty?

Myslim tim, jestli to dev/default nepatri spis nekde na SAPI clienta. Kdyz to trochu prezenu, `getStorageBranchType` mi porad dava smysl, `getEncryptorBranchType` uz tolik ne (mysleno getBranchType z domeny encryptoru, ne ze encryptor je branchovany :slightly_smiling_face:)

Vsechno, co ted zavisi na JobDefinition tim bude nutne zaviset na ObjectEncryptoru (vc. https://github.com/keboola/job-queue/blob/554828cd7a8c2a1e920515266a9626dd714dbd47/libs/job-configuration/src/JobDefinition/JobDefinition.php#L10)

Reply:
jo, ale prdím na to